pkg/core/storage/store.go

```diff
@@ -28,7 +28,7 @@ const (
 var ErrKeyNotFound = errors.New("key not found")
 
 type (
-	// Store is anything that can persist and retrieve the blockchain.
+	// Store is anything that can persist and retrieve the blockchain
```

Comment:
Separate commit.

Reply:
decided to leave fixing comments for later pr